### PR TITLE
Feature/add real twisted logging and file support

### PR DESCRIPTION
**Pull request by danaronson**

### CODE REVIEW

File: server.py [hunk 1]
```diff
@@ -231,4 +254,9 @@ def get_data_from_neighbors():
 
 site = twserver.Site(Simple())
 reactor.listenTCP(int(config.get('port', 8080)), site)
+
```
Comment (mitra42) [inline server.py:257]: Is this still true - or did I see a reset at line 72

File: tests/__init__.py [hunk 2]
```diff
@@ -105,7 +105,8 @@ def reset(self, delete_files = True):
         if delete_files:
             for file_name in os.listdir(self.directory):
                 file_path = os.path.join(self.directory, file_name)
-                if os.path.isfile(file_path):
+                if os.path.isfile(file_path) and (file_name not in ['log.txt']):
+                    logger.info('file path is %s' % file_path)
```
Comment (mitra42) [inline tests/__init__.py:109]: Should this be config['log_file_name']

File: tests/__init__.py [hunk 3]
```diff
@@ -157,8 +158,9 @@ def run_server(server_urls=None, port=None):
     with TemporaryDirectory() as tmp_dir_name:
         logger.info('created temporary directory %s' % tmp_dir_name)
         config_file_path = tmp_dir_name + '/config.ini'
-        config_data = '[DEFAULT]\nDIRECTORY = %s\nLOG_LEVEL = INFO\nPORT = %d\nTesting = True\n"BOUNDING_BOX_MINIMUM_DP = 2\nBOUNDING_BOX_MAXIMUM_SIZE = 0.001\nLOCATION_RESOLUTION = 4\nAPP_TESTING = 2.0\n' % (
-            tmp_dir_name, port)
+        log_file_path = tmp_dir_name + '/log.txt'
```
Comment (mitra42) [inline tests/__init__.py:161]: If this is a log_file_path why are we calling it log_file_name - that's confusing (I presume name is "log.txt" path is "/tmp/xxx/log.txt"

File: server.py (hunk 1 above)
Comment (danaronson) [reply server.py:257]: that one worked, this one doesn't.  i want to reset to allow the file to be moved, but not yet sure how.  i'll do that in another branch at some point (#108 )

File: tests/__init__.py (hunk 2 above)
Comment (danaronson) [reply tests/__init__.py:109]: nope, no such thing in testing... (this isn't the code)

File: tests/__init__.py (hunk 3 above)
Comment (danaronson) [reply tests/__init__.py:161]: i'll fix

**Pull request merged**